source/user-guide/harvesting/harvesting-simpleurl.rst

```diff
@@ -50,5 +50,7 @@ Adding a simple URL harvester
     - *pageSizeParam* : ``rows``
     - *toISOConversion* : ``DKAN-to-ISO19115-3-2018``
 
+    Note: In GN versions > 4.2.2 the ``recordIdPath`` property needs to start with a ``/``. Schemas bearing similar names can be selected via a dropdown here.
```

Comment:
Not a documenter expert, but if we start adding additional notes for each change in each release, readability of the doc in a couple of years will be a challenge, end users will have to read the top text and then applies all the notes with changes in each versions to have the full picture... Readability would be facilitated  if the main text represent the current status and additional note highlight changes in past version so end user will have to read the note only if using old stuff? Also consider https://geonetwork-opensource.org/manuals/4.0.x/en/contributing/writing-documentation.html#versionadded-versionchanged-and-deprecated

Reply:
I agree.
Breaking changes should be documented somewhere though.